### ISSUE TITLE
Error loading result from search on discovery

### ISSUE THREAD

**Issue by Lane** [Hyperobjekt/seda-site#30]
from sean:

> when i’m in discovery, and use the search bar, i can find districts but they don’t show up on charts. specifically looking at first discovery (affluent schools...) and last chart. more specifically, here (see below image). if i type in stockton higher up in the article, it works, but not right here
![image](https://user-images.githubusercontent.com/21034/65470724-99fc8780-de29-11e9-977a-76b78c4b01a1.png)

was able to replicate:
1. Go to relevant discovery article: https://edopportunity.org/discoveries/affluent-schools-are-not-always-best/
2. Scroll down to where state 14 activates
3. Search for "stockton unified", and select
4. The location does not get added to the chart, and the following error is logged: 

![image](https://user-images.githubusercontent.com/21034/65470690-7b968c00-de29-11e9-9168-24856fe1ce22.png)

**Comment by amygroshek** [comment]
Yeah I think I might know a way around this.

**Comment by amygroshek** [comment]
@Lane @james-minton There's a similar issue that occurs in the other article, article1.js, a lot of schools that appear in the search results don't get highlighted. A hit with data is returned from the scatterplot search component but they aren't highlighted when passed into the scatterplot. It's state6 and above, the charts that use wb_ses as the x-axis. My home district and several others of similar size don't get added: Adams-Friendship School District, Mauston School District, Baraboo School District, Wisconsin. All of these districts are highlighted in the previous dataset, which uses w_avg as the x-axis. This fix won't solve that issue, and if someone follows up with Sean about this they'll probably want to let him know so he doesn't think we just didn't fix it.

**Comment by amygroshek** [comment]
@james-minton Pushed branch fixup-30 with this fix.

**Comment by Lane** [comment]
@amygroshek do the hits that are returned have `wb_ses` values?  i know that metric is unavailable for a lot of places, so if that's the case it will not be present in the scatterplot.

**Comment by amygroshek** [comment]
@Lane That's correct, these hits don't return `wb_ses`. But Sean won't know the difference. Just wanted to mention it.
